web/src/lib.rs

```diff
@@ -224,14 +223,24 @@ impl Ruffle {
     }
 
     pub fn pause(&mut self) {
-        // Remove instance from the active list.
         INSTANCES.with(|instances| {
```

Comment:
I removed these two comments because they seem to be a copy/paste error from the `destroy` function.